client/src/pages/CreateNewPage.js

```diff
@@ -0,0 +1,22 @@
+import "./CreateNewPage.css";
+
+export default function CreateNewPage() {
+  return (
+    <div className="Form">
+      <div className="Page__heading">
+        <h2>Add a recipe</h2>
```

Comment:
is this maybe an h1? there is no h1 yet, right?

Reply:
Very good remark ! Now is edited ! Thanks ! :)